modules/atomic-widgets/base/atomic-widget-base.php

```diff
@@ -114,22 +115,10 @@ final public function get_stack( $with_common_controls = true ) {
 	}
 
 	final public function get_atomic_settings(): array {
-		$schema = static::get_props_schema();
-		$raw_settings = $this->get_settings();
-
-		$transformed_settings = [];
-
-		foreach ( $schema as $key => $prop ) {
-			if ( array_key_exists( $key, $raw_settings ) ) {
-				$transformed_settings[ $key ] = $raw_settings[ $key ];
-			} else {
-				$transformed_settings[ $key ] = $prop->get_default();
-			}
-
-			$transformed_settings[ $key ] = $this->transform_setting( $transformed_settings[ $key ] );
-		}
-
-		return $transformed_settings;
+		return Props_Handler::for_settings()->handle(
+			$this->get_settings(),
+			static::get_props_schema()
+		);
```

Comment:
do you think that passing an array would be more readable?

```suggestion
		return Props_Handler::for_settings()->handle( [
			'props' => $this->get_settings(),
			'schema' => static::get_props_schema(),
		] );
```

Reply:
Done